unstructured/file_utils/model.py

```diff
@@ -30,6 +66,9 @@ class FileType(enum.Enum):
     _alias_mime_types: tuple[str, ...]
     """MIME-types accepted as identifying this file-type."""
 
+    _partitioner_full_module_path: str | None
```

Comment:
This repo still supports 3.9 and I don't think this syntax exists in that version. 
```suggestion
    _partitioner_full_module_path: Optional[str]
```

Reply:
All the existing fields already used that style so I think we're okay